grpc-messenger-service/src/main/java/com/kucuk/server/service/MessageService.java

```diff
@@ -37,14 +28,7 @@ public void createMessage(CreateMessageRequest request,
                 @Override
                 public void completed(Response response) {
                     Long blockingCallTimeStamp = response.readEntity(Long.class);
-                    responseObserver.onNext(CreateMessageResponse.newBuilder()
-                            .setResponseId(Instant.now().toEpochMilli())
-                            .setHash(sha256hex)
-                            .setTime(blockingCallTimeStamp)
-                            .setSampleBooleanField(!request.getSampleBooleanField())
-                            .setSampleDoubleField(request.getSampleDoubleField() + 0.1)
-                            .setSampleIntegerField(request.getSampleIntegerField() + 1)
-                            .build());
+                    responseObserver.onNext(MessageServiceHelper.newCreateMessageResponse(request, blockingCallTimeStamp));
```

Comment:
let's not move creating new responses to helper classess. In a real service, business logic will happen here and create a response accordingly. So le't keep it closer to a real service even though it is repetitive. Helper classees are useful certain scenarios but they also makes harder to read code since antying can happen in a helper

Reply:
Moved the helper methods to MessageService instead. This would avoid duplication as well as it will be easier to read imo.